STYLE_GUIDE.md

```diff
@@ -1,3 +1,66 @@
 # Fortran stdlib Style Guide
 
-This document will describe the code style to use.
+> [A]utomate and codify as much as you possibly can while remembering that the human touch is still necessary to praise
+> and be kind to your contributors.
+> Let robots handle your project’s pedantry and humans handle your project’s empathy.
+
+-- @mikemcquaid, [Homebrew] project leader<sup>[1]</sup>
+
+[1]: https://mikemcquaid.com/2018/06/05/robot-pedantry-human-empathy/
+[Homebrew]: https://brew.sh
```

Comment:
Personally not crazy about this quote.

Reply:
Yeah, me neither.

I do want to automate formatting and everything else. But the quote doesn't quite get get excited.

Reply:
Haha, OK, that's fine with me. The point was just to convey a need for human empathy, and to highlight the fact to other maintainers that our time is better spent on substantive discussions. Where people have strong opinions about, e.g., style, then it's nice to have a robot complaining about it instead of a maintainer. Contributors tend to not take offense or want to argue, and everyone else gets to spend more time doing productive work. If you want to remove it, that's fine by me. BTW, I would recommend reading the article linked, and also another one about maintaining open source and the user-contributor-maintainer funnel (like a sales funnel).

Reply:
Here is the contributor funnel article, if you're interested: https://mikemcquaid.com/2018/08/14/the-open-source-contributor-funnel-why-people-dont-contribute-to-your-open-source-project/

And here is the article that the quote is from: https://mikemcquaid.com/2018/06/05/robot-pedantry-human-empathy/

I think both are definitely worth a read.